go.mod

```
@@ -47,21 +51,19 @@ require (
 	golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392 // indirect
 	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	golang.org/x/tools v0.0.0-20191025023517-2077df36852e // indirect
 	gopkg.in/yaml.v2 v2.2.7
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.0.0-20191016110408-35e52d86657a
-	k8s.io/apiextensions-apiserver v0.0.0-20191016113550-5357c4baaf65
-	k8s.io/apimachinery v0.0.0-20191028221656-72ed19daf4bb
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/code-generator v0.18.0-alpha.1.0.20191220033320-6b257a9d6f46
-	k8s.io/component-base v0.0.0-20191016111319-039242c015a9
-	k8s.io/kubectl v0.0.0-20191016120415-2ed914427d51
+	k8s.io/api v0.16.6
+	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783
```

```suggestion
	k8s.io/apiextensions-apiserver v0.16.6
```
Let's bump this dependency as well.

---

I have not updated it due to the warning message `../go/pkg/mod/k8s.io/component-base@v0.16.6/metrics/legacyregistry/registry.go:44:9: undefined: prometheus.InstrumentHandler` when running `go get k8s.io/apiextensions-apiserver@v0.16.6` 😬

---

Ouch! Okay, let's not bump this 😬